tests/test_api/test_types/test_chat_member.py

```diff
@@ -2,33 +2,39 @@
 
 from aiogram.api.types import ChatMember, User
 
-user = User(id=42, is_bot=False, first_name="User", last_name=None)
-
 
 class TestChatMember:
+    def setup(self):
+        self.__user = User(id=42, is_bot=False, first_name="User", last_name=None)
```

Comment:
It's just a good habit to use a private field when it is realy private. And no longer need to keep in mind where it can be used in the future, even for unit tests.

Reply:
ok